.gitignore

```diff
@@ -1,2 +1,3 @@
 target
-.idea
\ No newline at end of file
+.idea
```

Comment:
Is it worth adding:
.vscode & .mvn/

Reply:
We can add `.vscode`. What `.mvn` is for?

Reply:
I thought maven installed some files when ran via command line. But have just checked locally and can't see this happening. So happy to ignore .mvn. But Would be good to add .vscode as we have some developers who still use it.

Reply:
Updated